src/clj/puppetlabs/services/protocols/jruby_pool.clj

```diff
@@ -23,6 +23,10 @@
     [pool-context]
     "Unlocks the JRuby pool, allowing borrows to proceed.")
 
+  (worker-id
+    [pool-context instance]
+    "Returns a reference to a worker id (instance id or thread id).")
```

Comment:
Can you rephrase this a little to say "Returns the worker id for a given instance"? It's not really a reference we're returning, and since that word is used so much with the new pool, I'd rather not overload it.